hail/src/main/scala/is/hail/io/vcf/ExportVCF.scala

```diff
@@ -453,5 +460,52 @@ object ExportVCF {
         sb.result()
       }
     }.writeTable(ctx, path, Some(header), exportType = exportType)
+
+    if (tabix) {
+      exportType match {
+        case ExportType.CONCATENATED =>
+          info(s"Writing tabix index for $path")
+          TabixVCF(ctx.fsBc, path)
+        case ExportType.PARALLEL_SEPARATE_HEADER | ExportType.PARALLEL_HEADER_IN_SHARD =>
+          val localFsBc = ctx.fsBc
+          val files = fs.glob(path + "/part-*").map(_.getPath.getBytes)
+          info(s"Writing tabix index for $path")
+          ctx.backend.parallelizeAndComputeWithIndex(ctx.backendContext, files) { (pathBytes, _) =>
+            TabixVCF(localFsBc, new String(pathBytes))
+            null
+          }
+        case ExportType.PARALLEL_COMPOSABLE =>
+          warn("Writing tabix index for `parallel=composable` is not supported. No index will be written.")
+        case _ =>
```

Comment:
probably unnecessary -- not a user error, match error should be fine.

Reply:
Especially since if we get here, the mode has already been validated, removed.